pinery-miso/pom.xml

```diff
@@ -20,7 +20,7 @@
     <mysql.pw>tgaclims</mysql.pw>
     <db.params>
       autoReconnect=true&amp;characterEncoding=UTF-8&amp;allowPublicKeyRetrieval=true&amp;sslMode=DISABLED&amp;connectionTimeZone=SERVER&amp;cacheDefaultTimeZone=false</db.params>
-    <pinery.version>2.30.0</pinery.version>
+    <pinery.version>2.30.1-SNAPSHOT</pinery.version>
```

Comment:
Is this meant to be -SNAPSHOT?

Reply:
> Requires Pinery release before merging

it gets set to the new release before merge